recipes-core/systemd/systemd/aziot-identityd-startup-timeout-lte.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+TimeoutStartSec=180
```

Comment:
I don't have problems with setting this value globally, but I refrained from doing so, because other tests are obviously running fine with the much smaller timeout.
I know however that with the standard timeout of 60s (systemd default) the smoke test failed very often, even 120s weren't enough what perplexed me too to some extent, but apparently the identityd had several more attempts until it succeeded to fully start up.
in any case that meant to me (and still does) that this is specific to LTE for whatever reason.

Reply:
> still don't understand why we need this. as you wrote in the omnect-os PR **aziot-identityd should start after timesync and thus should have network**.

Reply:
ok, I found the root cause for the problem:

[root@hwt-rpi4-test ~]# systemctl list-dependencies --reverse systemd-networkd-wait-online.service 
systemd-networkd-wait-online.service
* `-network-online.target
*   |-aziot-edged.service
*   |-deviceupdate-agent.service
*   |-docker.service
*   |-iot-client-template-rs.service
*   `-omnect-device-service.service

this means that the identity service is lacking an important dependency, like we already suspected, in particular "Wants=network-online.target" afaics.
due to this lack the service is starting too early and it really depends on the LTE network how fast it connects to the internet whether the service can successfully start or not.
this also explains why we didn't see this problem via Ethernet uplink.

Reply:
moved condition to meta-omnect